rabbitmq-stream-dotnet-client.sln

```diff
@@ -9,6 +9,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tests", "Tests\Tests.csproj
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "RabbitMQ.Stream.Client.PerfTest", "RabbitMQ.Stream.Client.PerfTest\RabbitMQ.Stream.Client.PerfTest.fsproj", "{8EEF75BF-F0E5-493B-B642-40A4860733BA}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{208BA388-4CEE-2CBF-559C-5607C099C1F8}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Metrics", "docs\Metrics\Metrics.csproj", "{B5C8A5E0-18E8-44AB-B459-0B613F4A9EDD}"
```

Comment:
Can you remove `docs\Metrics\Metrics.csproj`.  rabbitmq-stream-dotnet-client.sln contains oly the package, test and perftest. `Docs` are outside